projects/plugins/jetpack/_inc/client/lib/plans/constants.js

```diff
@@ -416,6 +417,7 @@ export const JETPACK_FEATURE_PRODUCT_UPSELL_MAP = {
 	[ FEATURE_SIMPLE_PAYMENTS_JETPACK ]: PLAN_JETPACK_SECURITY_T1_YEARLY,
 	[ FEATURE_JETPACK_AI ]: PLAN_JETPACK_AI_YEARLY,
 	[ FEATURE_JETPACK_BOOST ]: PLAN_JETPACK_BOOST,
+	[ FEATURE_EMAIL_SUBSCRIPTION_JETPACK ]: PLAN_JETPACK_CREATOR_YEARLY,
```

Comment:
I'm actually not sure if this is required either 🤔  I think this is only used if the card is going to display an "Upgrade" button and I don't see that for Newsletter in the settings. Correct me if I am wrong 😄